purchases/src/main/kotlin/com/revenuecat/purchases/amazon/AmazonBilling.kt

```diff
@@ -580,9 +580,9 @@ internal class AmazonBilling(
         purchasesUpdatedListener?.onPurchasesFailedToUpdate(error)
     }
 
-    private fun checkObserverMode(): Boolean {
-        return if (observerMode) {
-            log(LogIntent.AMAZON_WARNING, AmazonStrings.WARNING_AMAZON_OBSERVER_MODE)
+    private fun finishingTransactionsIsDisabled(): Boolean {
```

Comment:
Nitpick but I would maybe rename to `shouldFinishTransactions` even if we have to negate the usages... This could also work but the naming makes me think of a property rather than a function, so maybe we can just convert it to a computed property instead.

Reply:
Yup, you're totally right. The idea behind the name was fluent if-statements: 
```kotlin
if (finishingTransactionsIsDisabled()) // then
```  
But it's better not to have negatives ("disabled") in names. Renamed!